### PR TITLE
Wk 783 suppress certain text on job cards

### PR DESCRIPTION
**Pull request by Kinjalpateloti**
### Summary by CodeRabbit

**Release Notes**

- **Refactor**: Simplified the job card display in the Vue.js templates, focusing on essential information like post title, organization, schedule, salary, and location.
- **Style**: Updated the theme configuration with a new version '3.0.0' for both light and dark themes.
- **Chore**: Removed the display of sector information from job cards.
- **Chore**: Removed the dollar sign prefix from salary values in job posts.
- **Chore**: Removed the location from the job card title if it is the default location.
- **Chore**: Removed the summary property and method from the Jobs class.

### CODE REVIEW

File: wp-content/themes/workingnyc/timber-posts/Jobs.php [hunk 1]
```diff
@@ -180,8 +180,7 @@ public function getOrganization() {
    * @return  String  Summary with HTML markup stripped
    */
   public function getSummary($limit = 120) {
```
Comment (SourabhChakraborty) [inline wp-content/themes/workingnyc/timber-posts/Jobs.php:182]: Where is this function used? Is it okay to remove it if it just returns an empty string?

File: wp-content/themes/workingnyc/views/jobs/job.twig [hunk 2]
```diff
@@ -11,8 +11,6 @@
       </a>
 
       <p class="c-card__subtitle text-alt">
-        {% if post.sector %}<b>{{ post.sector }}</b>{% endif %}
-        {% if post.sector and post.organization %}<span> {{ __('with', 'WNYC') }} </span>{% endif %}
         {% if post.organization %}<span>{{ post.organization }}</span>{% endif %}
       </p>
     </span>
```
Comment (SourabhChakraborty) [inline wp-content/themes/workingnyc/views/jobs/job.twig:16]: There's a section for the summary in this file and job.vue, can you remove it?

**Review by SourabhChakraborty — CHANGES_REQUESTED** [review]

File: wp-content/themes/workingnyc/timber-posts/Jobs.php (hunk 1 above)
Comment (Kinjalpateloti) [reply wp-content/themes/workingnyc/timber-posts/Jobs.php:182]: Yes we can remove it but it was calling from 2 places line number 81 and 119. If we need to remove then also we need to remove these 2 calls along with changes in job.twig.

File: wp-content/themes/workingnyc/views/jobs/job.twig (hunk 2 above)
Comment (Kinjalpateloti) [reply wp-content/themes/workingnyc/views/jobs/job.twig:16]: <div class="c-card__summary">
        <p v-html="post.context.summary"></p>
      </div>

Do I need to remove this div?

Comment (Kinjalpateloti) [reply wp-content/themes/workingnyc/views/jobs/job.twig:16]: <div class="c-card__summary">
        <p v-html="post.context.summary"></p>
      </div>

Do I need to remove this div also

File: wp-content/themes/workingnyc/timber-posts/Jobs.php (hunk 1 above)
Comment (SourabhChakraborty) [reply wp-content/themes/workingnyc/timber-posts/Jobs.php:182]: Yes, you should remove those lines. And you should also check where `summary` from lines 81 and 119 are being used and remove those as well.

File: wp-content/themes/workingnyc/views/jobs/job.twig (hunk 2 above)
Comment (SourabhChakraborty) [reply wp-content/themes/workingnyc/views/jobs/job.twig:16]: Yes you should remove the div

**Review by SourabhChakraborty — CHANGES_REQUESTED** [review]

**Pull request closed without merging**